synapse/storage/schema/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-SCHEMA_VERSION = 65  # remember to update the list below when updating
+SCHEMA_VERSION = 66  # remember to update the list below when updating
```

Comment:
this is needed, because adding the new columns will break compatibility with synapse versions before this PR, so we'll need to bump SCHEMA_COMPAT_VERSION.